bang/magicollections.py

```diff
@@ -1897,6 +1897,24 @@ def to_fields(self, view, item, *args, **kwargs):
         setSubField(fields, 'length', key='value', value=lambda f: item.length_in_minutes)
         setSubField(fields, 'unlock', key='value', value=item.unlock_sentence)
 
+        for difficulty, verbose_name in models.Song.DIFFICULTIES:
+            note_image = u'{}img/note.png'.format(RAW_CONTEXT['static_url'])
```

Comment:
Use `staticImageURL` function.

Reply:
I just saw that it was from my old code. Sorry, that's because I didn't have that function at the time!

Reply:
✅